Resources/doc/emails.rst

```diff
@@ -24,6 +24,36 @@ To enable it, update your configuration as follows:
             confirmation:
                 enabled: true
 
+Confirmation of Changed Email
+-----------------------------
+
+When a user changes her email address on the edit profile page,
+the bundle can send a confirmation email to the new address. The
+new address will only be activated / written to the database once
+the user clicks on the confirmation link in the confirmation email.
+
+Requiring email update confirmation is turned off by default.
+To enable it, update your configuration as follows:
```

Comment:
Is this true? I can't see the code where this is only enabled with the `email_update_confirmation` config - I don't see that config (the enabled part) used at all.